input/intro-notes/StructureDefinition-au-coverage-intro.md

```diff
@@ -2,3 +2,11 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When populating the type using the preferred ValueSet :
```

Comment:
You've correctly implemented the ticket:  "Value SetActCoverageTypeCode"
However, I think it's actually supposed to be "ValueSet ActCoverageTypeCode". Here's the value set at THO: http://terminology.hl7.org/ValueSet/v3-ActCoverageTypeCode